exo/orchestration/standard_node.py

```diff
@@ -346,6 +385,7 @@ async def periodic_topology_collection(self, interval: int):
         if DEBUG >= 2: print(f"{did_peers_change=}")
         if did_peers_change:
           await self.collect_topology()
+          await self.select_best_inference_engine()
       except Exception as e:
         print(f"Error collecting topology: {e}")
```

Comment:
I guess this should say "Error collecting topology or selecting best inference engine"